.github/workflows/docs.yml

```diff
@@ -21,15 +21,18 @@ on:
     # tags:
     #   - '*'
 
-permissions:
-  contents: read
+permissions: {}
 
 jobs:
   build:
     if: github.repository == 'sigstore/model-transparency'  # Don't do this in forks
     runs-on: ubuntu-latest
+    permissions:
+      contents: write # needed to upload artifact
```

Comment:
we don't currently have `contents: write` and the workflow works just fine

https://github.com/sigstore/model-transparency/actions/workflows/docs.yml